diff --git a/template.php b/template.php
index 1d59967..3446d21 100644
--- a/template.php
+++ b/template.php
@@ -366,7 +366,7 @@ function zurb_foundation_preprocess_field(&$variables) {
   // Add class to a specific fields across content types.
   switch ($variables['element']['#field_name']) {
     case 'body':
-      $variables['classes_array'][] = array('body');
+      $variables['classes_array'][] = 'body';
       break;
 
     case 'field_summary':
@@ -376,12 +376,12 @@ function zurb_foundation_preprocess_field(&$variables) {
     case 'field_link':
     case 'field_date':
       // Replace classes entirely, instead of adding extra.
-      $variables['classes_array'][] = array('text-content');
+      $variables['classes_array'] = array('text-content');
       break;
 
     case 'field_image':
       // Replace classes entirely, instead of adding extra.
-      $variables['classes_array'][] = array('image');
+      $variables['classes_array'] = array('image');
       break;
 
     default:
